SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -2302,6 +2335,33 @@ void ActorSystem::UpdateInterestComponent(AActor* Actor)
 	NetDriver->Connection->SendComponentUpdate(EntityId, &Update);
 }
 
+void ActorSystem::UpdateClientInterest(AActor* Actor, const bool bOverwrite)
+{
+	if (Actor->IsA(APlayerController::StaticClass()) && GetDefault<USpatialGDKSettings>()->bUseClientEntityInterestQueries)
+	{
+		Worker_CommandRequest CommandRequest{};
+		const bool bRequestValid = NetDriver->InterestFactory->CreateClientInterestDiff(Actor, CommandRequest, bOverwrite);
+
+		if (bRequestValid)
+		{
+			const Worker_EntityId SystemEntityId =
+				Cast<USpatialNetConnection>(Actor->GetNetConnection())->ConnectionClientWorkerSystemEntityId;
+
+			NetDriver->Connection->SendCommandRequest(SystemEntityId, &CommandRequest, RETRY_MAX_TIMES, {});
+
+			UE_LOG(LogActorSystem, Log, TEXT("Interest diff: worker entity id %lld"), SystemEntityId);
```

Comment:
Logged everytime we send interest, should be a Verbose maybe?
```suggestion
			UE_LOG(LogActorSystem, Verbose, TEXT("Interest diff: worker entity id %lld"), SystemEntityId);
```

Reply:
Yep sounds good - just had it log for now when debugging